docs/administration/installation.md

```diff
@@ -0,0 +1,220 @@
+
+The funding extension is still under development and there exists no stable release yet.
+
+In the following, `$DRUPAL_ROOT` refers to the drupal installation root, for example `/var/www/web`.
+
+## Status of this document
+
+The status of this document is **work in progress**.
+
+The installation instructions were used on a system with
+
+- CiviCRM 5.67.3
+- Drupal 10.1.6
+- PHP 8.1
+
+## Configure upload of private files
+
+```bash
+mkdir $DRUPAL_ROOT/drupal/web/sites/default/files/private
+chmod g+w $DRUPAL_ROOT/drupal/web/sites/default/files/private
+```
+
+Grant read and write permissions for the directory `private` to the webserver user.
+
+Set the following setting in `$DRUPAL_ROOT/drupal/web/sites/default/settings.php`:
+
+```php
+$settings['file_private_path'] = 'sites/default/files/private';
+```
+
+## Drupal modules and dependencies
+
+Edit `$DRUPAL_ROOT/drupal/composer.json` and add the following to repositories:
+
+```
+        "json_forms": {
+            "type": "vcs",
+            "url": "git@github.com:systopia/drupal-json_forms.git"
+        },
+        "custom/civiremote_funding": {
+            "type": "vcs",
+            "url": "git@github.com:systopia/drupal-civiremote_funding.git"
+        },
+        "custom/civiremote": {
+            "type": "vcs",
+            "url": "git@github.com:systopia/civiremote"
+        },
+```
+
+Open a terminal at `$DRUPAL_ROOT` and enter
+
+```bash
+composer require custom/civiremote_funding
+drush pm:enable civiremote_funding
+```
+
+!!! note
+    If CMRF Core is published as stable, `composer require drupal/cmrf_core:^2.1` is no longer required.
+
+## Further modules/dependencies
+
+- `fontawesome` is not required if Font Awesome is provided by other means.
+- `formtips` is not required, but recommended.
+- `symfony/property-access` is a dependency of the funding extension.
+
+```
+composer require drupal/fontawesome drupal/formtips symfony/property-access
+drush pm:enable fontawesome formtips
+```
+
+Enter this command, so that changes to views etc. can be applied:
+
+```
+composer require drupal/config_update
+drush pm:enable config_update
+```
+
+!!! note
+    The last step is no longer necessary as soon as we have releases of `civiremote_funding` with update routines.
+
+### Configure fontawesome
+
+You can optionally change the settings to provide Font Awesome files locally:
+
+```
+drush fa:download
+```
+
+Open `/admin/config/content/fontawesome` and uncheck the option **Use external file (CDN) / local file?**
+
+
+### Configure formtips
+
+Set the following under `/admin/config/user-interface/formtips` (adjust times if necessary):
+
+```
+Trigger Action: Hover
+Selector: :not(#funding-form *)
+Interval: 100
+Timeout: 100
+```
+
+The above selector restricts the module to the funding forms.
+
+## Install CiviCRM Extensions
+
+Install the following extensions, use the newest release if not otherwise indicated:
```

Comment:
In this case there is a stable release 1.3.1 that is from september 2023 and alpha releases of 1.4. Does your comment refer to the fact that we need 1.4 despite it being alpha releases? Because I don't think that the last stable release (1.3.1) seems to be too old in this case.

I added your suggestion. A similar case would be civibanking (most recent tag 1.0-alpha1) and civioffice (most recent tag 1.0-beta2). Shall I add those two versions as well?

Reply:
IIRC there was an issue with version 1.3 of identitytracker.

Yes, I think it makes sense to start with version 1.0 of civibanking and civioffice.